File: Python/BEGINNER/2003 Sunday Morning.py
# SUBPROGRAM
# MAIN PROGRAM
while True:
    try:
        arr = list(input().split())
        hours = int(arr[0][0])
        minutes = int((arr[0][2] + arr[0][3]))
        minutes = int(minutes)
        if ((hours <= 7) and (minutes == 0)) or (hours < 7):
            print("Atraso maximo: 0")
        else:
            if hours == 9:
                print("Atraso maximo: %d" % (60*2))
            elif hours == 8:
                print("Atraso maximo: %d" % (60+minutes))
            else:
                print("Atraso maximo: %d" % minutes)
    except EOFError:
        break

File: C/BEGINNER/2003 Sunday Morning.c
#include <stdio.h>

int main()
{
    int hours, minutes;
    char colon;
    while (scanf("%d %c %d", &hours, &colon, &minutes) != EOF)
    {
        int delay = 0;
        if (((hours <= 7) && (minutes == 0)) || (hours < 7))
        {
            printf("Atraso maximo: 0\n");
        }
        else
        {
            if (hours == 9)
                printf("Atraso maximo: %d\n", (60 * 2));
            else if (hours == 8)
                printf("Atraso maximo: %d\n", (60 + minutes));    
            else
                printf("Atraso maximo: %d\n", (minutes));
        }
    }
    return 0;
}
